x-pack/test/functional/services/ml/api.ts

```diff
@@ -29,6 +28,16 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 export type MlApi = ProvidedType<typeof MachineLearningAPIProvider>;
 
 type ModelType = 'regression' | 'classification';
+export type TrainedModelName =
```

Comment:
It would make more sense to me if these hardcoded names lived next to the hardcoded `tinyTrainedModels` array.

Reply:
Changed the definition of supported models and the name type in [a063134](https://github.com/elastic/kibana/pull/136010/commits/a0631345129027863cf57ae5cc793f1c799440d9).